multinode-demo/start_nodes.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Format for the nodes file
+# node1 public-ip|node1 privte-ip
+# node2 public-ip|node2 privte-ip
+# ....
+# e.g.
+# 35.197.92.75|10.138.0.6
+# 35.185.232.222|10.138.0.3
+# 35.233.137.53|10.138.0.5
```

Comment:
instead of a file format you have to parse with an associative array, just put the associative array in the config file and source it...

e.g.

nodes_file:

ip_addrs = ( [35.197.92.75]=10.138.0.6
                    [35.185.232.222]=10.138.0.3
                    ....
            )
```

then in start_nodes.sh:
```
. "${nodes_file}"
```